src/Files.Uwp/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -99,6 +99,8 @@ public string SpaceText
 
         public SectionType Section { get; set; }
 
+        public SidebarContextMenuOptions MenuOptions { get; set; }
```

Comment:
Does this need to reference the sidebar, or can it be named `ContextMenuOptions`?

Reply:
> Does this need to reference the sidebar, or can it be named `ContextMenuOptions`?
No, it doesn't have to be called that, I can change it.